README.md

```diff
@@ -136,7 +103,22 @@ class Product {
 ```
 
 Afterwards you are able to use the created entity as if you were using a database.
-Let's assume we have used the value ```http://www.yourSite.com/api/products``` for the product entity's ```@Table``` annotation.
+
+By using this setting the driver is doing a lot of stuff by itself:
```

Comment:
das hier is zu unauffällig - eigtl will ich ja wissen: Ok, das is einfach, aber dick un fett: Nur wenn ich folgende prerequisites einhalte